src/logger.rs

```diff
@@ -0,0 +1,54 @@
+extern crate chrono;
+
+use chrono::Utc;
+use chrono::prelude::DateTime;
+
+use std::time::SystemTime;
+use termion::{color, style};
+use std::fmt;
+
+pub enum Severity {
+    Error,      // There is a problem that causing the program to be unstable to be unable to run properly
+    Warning,    // There is a problem that might cause the program to be unstable or to be unable to run properly
+    Log,        // There is information to note
+}
+
+pub trait Interface {
+    fn log(&mut self, msg: &str, sev: Severity);
+}
+
+pub struct Log {
+    pub message: String,
+    pub severity: Severity,
+    pub timestamp: SystemTime,
+}
+
+impl fmt::Display for Log {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+
+        let severity = match &self.severity {
+            Severity::Error => format!("{}{}error:{}", color::Fg(color::Red), style::Bold, style::Reset),
```

Comment:
Oh! Add a tab to make these extra nice.
Does it look nicer? I can't actually tell
![image](https://user-images.githubusercontent.com/35516367/133026564-a0b2e8c5-2ea5-46b5-999d-dea3452cc07b.png)